legal-api/src/legal_api/resources/v2/business/business.py

```diff
@@ -167,56 +166,45 @@ def search_businesses():
     try:
         json_input = request.get_json()
         identifiers = json_input.get('identifiers', None)
+        search_filter_name = json_input.get('name', None)
+        search_filter_type = json_input.get('type', None)
+        search_filter_status = json_input.get('state', None)
+        page = json_input.get('page', 1)
+        limit = json_input.get('limit', 100)
+
+        try:
```

Comment:
Are there other pagination functions? Might be a good idea to wrap this into a function?